src/core/OmniInputStories.ts

```diff
@@ -264,6 +264,53 @@ const App = () => <${toPascalCase(tagName)}${args.label ? ` label='${args.label}
     return Clearable;
 };
 
+export const CustomClearableSlotIcon = <T extends HTMLElement, U extends BaseArgs>(
```

Comment:
Dont think Icon needs to be part of the name here. Anything can be slotted, not necessarily an icon.

Reply:
Updated to be named CustomClearableSlot